scripts/run-benchmarks-on-travis.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -v
+
+BASE_BRANCH=master
+
+if [[ $TRAVIS == true ]]; then
+    if [[ $TRAVIS_PULL_REQUEST == true ]]; then
+        CURRENT_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+        exit 0 # avoid double run on commit and PR
+    else
+        CURRENT_BRANCH=$TRAVIS_BRANCH
+    fi
+else
+    CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+fi
+        
+COMMIT_MSG=$(git show -s --format="%s")
+
+SANTI_BR_NAME=$(echo $CURRENT_BRANCH | sed 's/\W/_/g')
+COMMIT_SHA=$(git rev-parse  HEAD)
+TIME=$(date +%Y%m%d%H%M%S)
+BM_JOB_NAME="BMCI-${SANTI_BR_NAME}-${COMMIT_SHA:0:7}-${TIME}"
+
+if [[ $COMMIT_MSG != *"[bm]"* ]]; then
```

Comment:
I think we can run benchmarking by default. The user can use “si” to skip CI together with benchmarking if needed.